src/Interactive/EditorFeatures/Core/Implementation/Interactive/Commands/LoadCommand.cs

```diff
@@ -9,10 +9,12 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.InteractiveWindow;
 using Microsoft.VisualStudio.InteractiveWindow.Commands;
+using Microsoft.VisualStudio.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Interactive
 {
     [Export(typeof(IInteractiveWindowCommand))]
+    [ContentType("code")]
```

Comment:
Note:  hard-coded constant intentional...this class will be removed soon, and it didn't seem worth it to add a public constant for the other two cases.